<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpBox1_Rotor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Вращение</string>
    </property>
    <widget class="QLabel" name="Rtr_lbl01_Speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Скорость</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rtr_lEd1_Speed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rtr_lEd3_Step">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rtr_lbl03_Step">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rtr_pBtn2_Backward">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Против часовой</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rtr_pBtn4_Step_Bkwd">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг против часовой</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rtr_lbl04_mm">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>мм</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Rtr_lEd2_DiamNum">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>139</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rtr_lbl05_mm">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>°</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rtr_pBtn3_Step_Fwd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг по часовой</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rtr_pBtn1_Forward">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>По часовой</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rtr_lbl02_Diam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Диаметр</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rtr_pBtn6_Stop">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>450</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rtr_pBtn5_Home">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Домой</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBox2_Data">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>201</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Данные с энкодера и датчиков</string>
    </property>
    <widget class="QLabel" name="Data_lbl12_ZstepData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl02_BxData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bx</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl06_BzData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bz</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl19_T">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>T</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl17_PhiStep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>70</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг по Phi</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl20_TData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>T</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl14_PhiData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>330</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Data_txtBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl01_Bx">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Bx</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl07_Z">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl08_ZData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl15_PhiErr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Phi error</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl16_PhiErrData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phi error</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl18_PhiStepData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Data_pBtn_ShowData">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать данные
 с датчиков</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl11_Zstep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг по Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl10_ZerrData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z error</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl04_ByData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>By</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl05_Bz">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Bz</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl03_By">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>By</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl09_Zerr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>70</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>Z error</string>
     </property>
    </widget>
    <widget class="QLabel" name="Data_lbl13_Phi">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Phi</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBox3_Line">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>321</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Линейка</string>
    </property>
    <widget class="QLabel" name="Line_lbl2_Step">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг</string>
     </property>
    </widget>
    <widget class="QLabel" name="Line_lbl1_Speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Скорость</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Line_pBtn1_Forward">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Вниз</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Line_pBtn6_Stop">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>450</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Line_lEd1_Speed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Line_lEd2_Step">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Line_pBtn3_Step_Fwd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг вниз</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Line_pBtn4_Step_Bkwd">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>320</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг вверх</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Line_pBtn5_Home">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Домой</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Line_pBtn2_Backward">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Вверх</string>
     </property>
    </widget>
    <widget class="QLabel" name="Line_lbl2_Step_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>мкм</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
